conferences/ticketing_storage.go

```diff
@@ -0,0 +1,605 @@
+func updateConferenceSlot(ctx context.Context, tx *sqldb.Tx, cslot *ConferenceSlot, conferenceID int64) error {
+	var sqlStatement = `UPDATE conference_slot 
+	SET conference_id = $1, name = $2, descripcion = $3, cost =$4, capacity=$5, 
+	start_date = $6, end_date = $7, purchaseable_form = $8, purchaseable_until = $9, 
+	available_to_public $10, depends_on_id = NULL
+	WHERE id = $11`
+	var args = []interface{}{
+		conferenceID,
+		cslot.Name,
+		cslot.Description,
+		cslot.Cost,
+		cslot.Capacity,
+		cslot.StartDate,
+		cslot.EndDate,
+		cslot.PurchaseableFrom,
+		cslot.PurchaseableUntil,
+		cslot.AvailableToPublic,
+	}
+	/* if e.DependsOn != nil {
+		sqlStatement = `UPDATE conference_slot
+	SET conference_id = $1, name = $2, descripcion = $3, cost =$4, capacity=$5,
+	start_date = $6, end_date = $7, purchaseable_form = $8, purchaseable_until = $9,
+	available_to_public $10, depends_on_id = $11
+	WHERE id = $12`
+		args = append(args, e.DependsOn.ID)
+	} */
+	args = append(args, cslot.ID)
+
+	res, err := exec(ctx, tx, sqlStatement, args...)
+
+	if err != nil {
+		return fmt.Errorf("updating conference slot: %w", err)
+	}
+	ra, err := res.RowsAffected()
+	if err != nil {
+		return fmt.Errorf("updting conference slot: %w", err)
+	}
+	if ra == 0 {
+		return fmt.Errorf("slot was not updated")
+	}
+
+	return nil
+}
+
+// createSlotClaim saves a slot claim and returns it with the populated ID
+func createSlotClaim(ctx context.Context, tx *sqldb.Tx, slotClaim *SlotClaim) (*SlotClaim, error) {
+	var err error
+	// FIXME: Add a check for capacity not exceeded on conference.
+	for i := 0; i < 3; i++ {
+		row := queryRow(ctx, tx,
+			`INSERT INTO slot_claim (ticket_id, redeemed, conference_slot_id) VALUES ($1, $2, $3)
+		RETURNING id, ticket_id, redeemed`,
+			slotClaim.TicketID, slotClaim.Redeemed, slotClaim.ConferenceSlot.ID)
+
+		results := SlotClaim{}
+		err = row.Scan(&results.ID, &results.TicketID, &results.Redeemed)
+
+		if err != nil {
+			return nil, fmt.Errorf("saving slot claim: %w", err)
+		}
+
+		results.ConferenceSlot = slotClaim.ConferenceSlot
+		return &results, nil
+	}
+	return nil, fmt.Errorf("failed to insert: %w", err)
+}
+
+const (
+	tableAttendee               = "attendee"
+	tableAttendeeSlotClaims     = "attendee_to_slot_claims"
+	slotClaimIDUniqueConstraint = "slot_claim_id_is_unique"
+)
+
+// updateAttendee saves the passed attendee attributes on top of the existing one.
+func updateAttendee(ctx context.Context, tx *sqldb.Tx, attendee *Attendee) (*Attendee, error) {
+	res, err := exec(ctx, tx,
+		`UPDATE attendee SET email = $1, coc_accepted = $2 WHERE id = $3`,
+		attendee.Email, attendee.CoCAccepted, attendee.ID)
+
+	if err != nil {
+		return nil, fmt.Errorf("updating attendee: %w", err)
+	}
+
+	ra, err := res.RowsAffected()
+	if err != nil {
+		return nil, fmt.Errorf("updating attendee: %w", err)
+	}
+	if ra == 0 {
+		return nil, fmt.Errorf("attendee was not updated")
```

Comment:
If no rows were updated, then there isn't someone who exists for that ID so I think this could be a better error message:
```suggestion
		return nil, fmt.Errorf("no such attendee")
```